File: KITTY/KURS_1/JAVA/DAY_30_1N_ALERTDIALOG/app/src/main/java/com/example/velic/day_30_1_alertdialog/MyDialogFragment2.java
package com.example.velic.day_30_1_alertdialog;

import android.app.AlertDialog;
import android.app.Dialog;
import android.content.DialogInterface;
import android.os.Bundle;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.fragment.app.DialogFragment;

public class MyDialogFragment2 extends DialogFragment {
    @NonNull
    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        String title = "Выбор есть всегда";
        String message = "Выбери пищу";
        String button1String = "Вкусная";
        String button2String = "Здоровая";

        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
        builder.setTitle(title);  // заголовок
        builder.setMessage(message); // сообщение
        builder.setPositiveButton(button1String, new DialogInterface.OnClickListener() {
            public void onClick(DialogInterface dialog, int id) {
                Toast.makeText(getActivity(), "Вы сделали правильный выбор",
                        Toast.LENGTH_LONG).show();
            }
        });
        builder.setNegativeButton(button2String, new DialogInterface.OnClickListener() {
            public void onClick(DialogInterface dialog, int id) {
                Toast.makeText(getActivity(), "Возможно вы правы", Toast.LENGTH_LONG)
                        .show();
            }
        });
        builder.setCancelable(true);

        return builder.create();
    }
}

File: TEST/DAY_30_1_ALERT_Dialog_kt/app/src/main/java/com/example/day_30_1_alert_dialog_k/MyDialogFragment2.kt
package com.example.day_30_1_alert_dialog_k

import android.app.AlertDialog
import android.app.Dialog
import android.os.Bundle
import android.widget.Toast
import androidx.fragment.app.DialogFragment
// пример создания диалогового окна с двумя кнопками на основе иллюстрации
class MyDialogFragment2 : DialogFragment() {
    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        val title = "Выбор есть всегда"
        val message = "Выбери пищу"
        val button1String = "Вкусная"
        val button2String = "Здоровая"

        val builder = AlertDialog.Builder(activity)
        builder.setTitle(title)  // заголовок
        builder.setMessage(message) // сообщение
        builder.setPositiveButton(button1String) { dialog, id ->
            Toast.makeText(activity, "Вы сделали правильный выбор",
                    Toast.LENGTH_LONG).show()
        }
        builder.setNegativeButton(button2String) { dialog, id ->
            Toast.makeText(activity, "Возможно вы правы", Toast.LENGTH_LONG)
                    .show()
        }
        builder.setCancelable(true)

        return builder.create()
    }
}
